WAW.API/Job/Resources/OfferResource.cs

```diff
@@ -0,0 +1,10 @@
+namespace WAW.API.Job.Resources;
+
+public class OfferResource {
+  public long Id { get; set; }
+  public string? Title { get; set; }
+  public string? Image { get; set; }
+  public string? Description { get; set; }
+  public string? SalaryRange { get; set; }
```

Comment:
Same thing as in the model. Only optional properties should be marked as nullable.